src/frontend/Komponenter/Migrering/MigrerFagsak.tsx

```diff
@@ -110,7 +132,13 @@ const MigrerFagsak: React.FC<{
 
     const migrerFagsak = () => {
         settMigrertStatus(byggHenterRessurs());
-        axiosRequest<string, void>(migrerFagsakConfig).then((res: Ressurs<string>) => {
+        axiosRequest<string, { ignorerFeilISimulering?: boolean }>({
+            method: 'POST',
+            url: `/familie-ef-sak/api/migrering/${fagsakPersonId}`,
+            data: {
+                ignorerFeilISimulering,
```

Comment:
Denne kan bli "undefined" nå - funker det da med backend? Burde den være default false istedenfor?

Reply:
Ja, backend funker med det, for å kunne støtte bakåtkompatibilitet :)